packages/components/src/components/ScalarFloating/ScalarFloating.vue

```diff
@@ -76,15 +76,17 @@ const { floatingStyles, middlewareData } = useFloating(targetRef, floatingRef, {
     v-if="isOpen"
     :disabled="!teleport"
     to="body">
-    <div
-      ref="floatingRef"
-      class="relative z-overlay"
-      :style="floatingStyles">
-      <slot
-        :data="middlewareData"
-        :height="targetHeight"
-        name="floating"
-        :width="targetWidth" />
+    <div class="scalar-app">
```

Comment:
wondering if using scalar-components would make sense here or not

Reply:
I could but we would then have to add more classes to the reset and stuff (probably not worth it).

Reply:
the system currently works!